orchestra/models/communication/models.py

```diff
@@ -102,7 +102,8 @@ class RequestCause(ChoicesEnum):
         RESTAFF = 'restaff'
 
     class Status(ChoicesEnum):
-        PROCESSING = 'processing'
+        SENDING_INQUIRIES = 'sending inquiries'
+        WAITING_FOR_RESPONSES = 'waiting for responses'
         COMPLETE = 'complete'
```

Comment:
COMPLETE used to mean "i'm done sending inquiries" but now it means "this is no longer an open request" -> what will that mean for existing COMPLETE requests that are still open? they will presumably disappear from the available tasks list.

you could consider renaming COMPLETE to FINISHED_SENDING_INQUIRIES and then adding a COMPLETE state?